st2common/tests/unit/services/test_workflow.py

```diff
@@ -32,8 +32,10 @@
 from st2common.exceptions import action as action_exc
 from st2common.models.db import liveaction as lv_db_models
 from st2common.models.db import execution as ex_db_models
+from st2common.models.db.pack import ConfigDB
```

Comment:
Please stay consistent with imports for this module.  Change to `from st2common.models.db import pack as pk_db_models` and move this line so the import is alphabetical.